content/en/opentelemetry/integrations/log_collection.md

```diff
@@ -82,6 +82,17 @@ filelog:
 
 {{< /tabs >}}
 
+### Custom tags
+In order to add custom Datadog tags to logs, set the `ddtags` attribute on the logs. For example, this can be done with the [transform processor][3]:
+```yaml
+processors:
+  transform:
+    log_statements:
+      - context: log
+        statements:
+          - set(attributes["ddtags"], "first_custom:tag, second_custom:tag")
```

Comment:
Just making sure that this is correct and that each custom tag shouldn't be in its own set of quotation marks

Reply:
Yes those are comma-separated